integreat_compass/locale/de/LC_MESSAGES/django.po

```diff
@@ -449,12 +449,39 @@ msgid "Home"
 msgstr "Start"
 
 #: cms/templates/index.html
-msgid "Arriving, reimagined."
-msgstr "Ankommen weitergedacht."
+msgid "Filter"
+msgstr "Filter"
+
+#: cms/templates/index.html
+msgid "Filter all offers"
+msgstr "Alle Maßnahmen filtern"
+
+#: cms/templates/index.html
+msgid "Search"
+msgstr "Suchen"
 
 #: cms/templates/index.html
-msgid "Discover Integreat Compass"
-msgstr "Intgreat Kompass entdecken"
+msgid "Categories"
+msgstr "Kategorien"
+
+#: cms/templates/index.html
+msgid "Submit"
+msgstr "Senden"
+
+#: cms/templates/index.html
+msgid "Offers"
+msgstr "Maßnahmen"
+
+#: cms/templates/index.html
+#, python-format
+msgid ""
+"\n"
+"                        %(amount_of_offers)s offers\n"
+"                        "
+msgstr ""
+"\n"
+"                        %(amount_of_offers)s Maßnahmen\n"
+"                        "
```

Comment:
In order to avoid these weird spaces, please use `trimmed` on the translation tag.

Reply:
Thanks! This also resolved itself, but that was very helpful for other PRs!